app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.error import WrongCommandError, SameNameError, ArgumentsError
+
+
+def copy_file(command: str) -> None:
+
+    command = command.split(" ")
+
+    if command[0] != "cp":
+        raise WrongCommandError("Incorrect command")
+
+    if command[1] == command[2]:
+        raise SameNameError("The copy should be renamed")
```

Comment:
https://github.com/mate-academy/py-copy-file#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.